jsonnetfile.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "dependencies": [
+        {
+            "source": {
+                "git": {
+                    "remote": "https://github.com/bitnami-labs/kube-libsonnet"
+                }
+            },
+            "version": "v1.14.6"
+        }
+    ],
+    "legacyImports": true
+}
```

Comment:
newline

Reply:
That is unfortunately modified by `jb install`, so I just left it as generated, otherwise there's a diff in every PR regarding the newline